frontend/src/components/CreateMemo/components/Locations/SavedLocations.tsx

```diff
@@ -1,41 +1,29 @@
 //@ts-nocheck
```

Comment:
remove this an all other mentions of `//@ts-nocheck` in the files you've changed